examples/core/gltf/app.js

```diff
@@ -169,20 +169,23 @@ export default class AppAnimationLoop extends AnimationLoop {
     return INFO_HTML;
   }
   constructor(opts = {}) {
-    super(opts);
+    super({
+      ...opts,
+      glOptions: {
+        // Use to test gltf with webgl 1.0 and 2.0
+        webgl1: true,
+        webgl2: true,
```

Comment:
This will always create webgl2 context if the platform supports it, is that what you want...

Reply:
Yes.